lib/sql-filter.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+/* global JuttleAdapterAPI */
+let FilterJSCompiler = JuttleAdapterAPI.compiler.ASTVisitor;
```

Comment:
Wouldn't FilterASTVisitor be a more appropriate name for this? JuttleAdapterAPI.compiler also exports a FilterJSCompiler so the name is misleading.